### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by learnwithalfred**
## Completed Project requirements

- Add the following validations: ✔️ 
    - For the User model:
        - Name must not be blank.
        - PostsCounter must be an integer greater than or equal to zero.
    - For the Post model:  ✔️ 
        - Title must not be blank.
        - Title must not exceed 250 characters.
        - CommentsCounter must be an integer greater than or equal to zero.
        -  LikesCounter must be an integer greater than or equal to zero.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @learnwithalfred,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly collaborate with your coding partner for this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by learnwithalfred** [comment]
Please did you check the project correctly?
project

This is the project I am submitting. It is a solo project

These are the requirements
req

Nowhere was it said that I should do the project with any partner.

File: spec/models/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'comment should have valid attributes' do
+    post = FactoryBot.create(:post)
+    user = FactoryBot.create(:user)
+    comment = FactoryBot.build(:comment, author_id: user.id, post_id: post.id)
```
Comment (codepantha) [inline spec/models/comment_spec.rb:7]: - [ ] All your tests are running successfully as shown in the attached image below. Kindly take a look and try to fix it 👍 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @learnwithalfred 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Done model validation
✔️ No linter errors
✔️ Good commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @learnwithalfred , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

### Highlights:
 ✔️ All specs are passing 
 ✔️ All linters are passing
 ✔️ Correct gitflow
 ✔️ Professional documentation

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

image

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@shoirata] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by learnwithalfred** [comment]
Thank you

**Pull request merged**